packages/amplify-graphql-searchable-transformer/src/__tests__/__snapshots__/amplify-graphql-searchable-transformer.tests.ts.snap

```diff
@@ -357,14 +392,49 @@ input SearchablePostSortInput {
   direction: SearchableSortDirection
 }
 
+input SearchablePostAggregationInput {
+  name: String!
+  type: SearchableAggregateType!
+  field: String!
+}
+
+enum SearchableAggregateType {
+  terms
+  avg
+  min
+  max
+  sum
+}
+
 type SearchablePostConnection {
   items: [Post]
   nextToken: String
   total: Int
+  aggregateItems: [SearchableAggregateResult]
 }
 
+type SearchableAggregateResult {
+  name: String!
+  result: SearchableAggregateGenericResult
+}
+
+type SearchableAggregateScalarResult {
+  value: Float!
+}
+
+type SearchableAggregateBucketResult {
+  buckets: [SearchableAggregateBucketResultItem]
+}
+
+type SearchableAggregateBucketResultItem {
+  key: String!
+  doc_count: Int!
```

Comment:
The snake case here looks inconsistent with the rest of the API. The RFC uses `count`.

Reply:
Open search returns this field as doc_count. I think it is better to leave these properties in the same names as returned by Opensearch so it is easy to extend for other aggregate functions in the future.

Reply:
How much effort would it be to perform the conversion?

cc: @renebrandel

Reply:
It would probably be 1-day effort. Importantly when we plan to support more OpenSearch aggregation methods in the future, we need to do conversion for different response data structures.